openshift/config/README.adoc

```diff
@@ -27,7 +27,23 @@ If your private config source provides agnosticv-operator configuration you may
 ansible-playbook setup-playbook.yaml \
 -e k8s_config_environment_level=dev \
 -e k8s_api_validate_certs=false \
--e babylon_private_repo=git@github.com:redhat-gpe/babylon-private.git \
```

Comment:
we probably even don't need to specify `private`. if `key_file_content` is present, create the key and pass  its path to `git` module as `key_file`

Reply:
I was trying to keep it simpler for the playbook invocation, avoiding needing to pass structured data... but I think you're right.

Reply:
Also, I used a path to an SSH key as the variable instead of file content because passing content forces the setup-playbook.yaml to have to write the key file content back out to a temporary file to be used by the `git` module.

Reply:
Oh, another reason I didn't pass a list is the list used when running the `setup-playbook.yaml` is that it is useful for rapid testing to have the babylon repo be local rather than pulling from git, but we're also bootstrapping the `anarchy-k8s-config` configuration and we need to create a list there... so the lists would need to be manipulated in some manner...

https://github.com/redhat-cop/babylon/blob/add-babylon-private-repo/openshift/config/common/templates/anarchy-k8s-config/anarchysubjects/babylon.yaml.j2#L16-L37

https://github.com/redhat-cop/babylon/blob/add-babylon-private-repo/openshift/config/setup-playbook.yaml#L14-L27

... and with all the power built in to `k8s_config` to load configuration from other sources, I don't think anyone will need more than a public and a private config repository.

Reply:
So yeah... I'm not entirely satisfied with the way it is, but I think making it a list makes it more complicated. I think best to let it be for now and wait to hear from users what their needs turn out to be.